packages/account/src/Sections/Security/Passwords/passwords.tsx

```diff
@@ -38,19 +42,8 @@ const Passwords = observer(() => {
 
     return (
         <div className='account__passwords'>
-            <DerivEmail
-                email={email}
-                social_identity_provider={social_identity_provider}
-                is_social_signup={is_social_signup}
-            />
-            <DerivPassword
-                email={email}
-                is_dark_mode_on={is_dark_mode_on}
-                is_social_signup={is_social_signup}
-                is_eu_user={is_eu_user}
-                financial_restricted_countries={financial_restricted_countries}
-                social_identity_provider={social_identity_provider}
-            />
+            <DerivEmail email={email} />
+            <DerivPassword email={email} />
```

Comment:
@likhith-deriv I believe `email` here is prop drilling, We can just get it from `useStore` if I'm not mistaken 🤔